src/main/java/eu/arrowhead/blacklist/service/dto/BlacklistQueryRequestDTO.java

```diff
@@ -0,0 +1,16 @@
+package eu.arrowhead.blacklist.service.dto;
+import java.util.List;
+
+import eu.arrowhead.blacklist.service.dto.enums.Mode;
+import eu.arrowhead.dto.PageDTO;
+
+public record BlacklistQueryRequestDTO(
+		PageDTO pagination,
+		List<String> systemNames,
+		Mode mode,
```

Comment:
should be string here, validation should check the value of mode is parseable as enum